build_tools/generate_authors_table.py

```diff
@@ -34,73 +48,121 @@ def group_iterable(iterable, size):
 
 def get_contributors():
     """Get the list of contributor profiles. Require admin rights."""
-    # get members of scikit-learn teams on GitHub
-    members = []
+    # get members of scikit-learn core-dev on GitHub
+    core_devs = []
     team = 11523
     for page in [1, 2]:  # 30 per page
-        reply = requests.get(
-            "https://api.github.com/teams/%d/members?page=%d"
-            % (team, page), auth=auth)
-        reply.raise_for_status()
+        reply = get("https://api.github.com/teams/%d/members?page=%d" %
+                    (team, page))
+        core_devs.extend(reply.json())
+
+    # get members of scikit-learn on GitHub
+    members = []
+    for page in [1, 2]:  # 30 per page
+        reply = get(
+            "https://api.github.com/orgs/scikit-learn/members?page=%d" %
+            (page, ))
         members.extend(reply.json())
 
     # keep only the logins
-    logins = [c['login'] for c in members]
-    # remove duplicate
-    logins = set(logins)
+    core_devs = [c['login'] for c in core_devs]
+    members = [c['login'] for c in members]
+
+    # add missing contributors with GitHub accounts
+    members.extend(['dubourg', 'mbrucher', 'thouis', 'jarrodmillman'])
+    # add missing contributors without GitHub accounts
+    members.extend(['Angel Soler Gollonet'])
+    # remove CI bots
+    members.remove('sklearn-ci')
+    members.remove('sklearn-lgtm')
+    members.remove('sklearn-wheels')
+
+    # remove duplicate, and get the difference of the two sets
+    core_devs = set(core_devs)
+    members = set(members)
+    emeritus = members.difference(core_devs)
 
     # get profiles from GitHub
-    profiles = [get_profile(login) for login in logins]
+    core_devs = [get_profile(login) for login in core_devs]
+    emeritus = [get_profile(login) for login in emeritus]
+
     # sort by last name
-    profiles = sorted(profiles, key=key)
+    core_devs = sorted(core_devs, key=key)
+    emeritus = sorted(emeritus, key=key)
```

Comment:
Or rather why doesn't this sort by last name as expected?

Reply:
what do you mean? It is ordered by last name, except composed last names such as Joris, Jeremy or myself.

Reply:
Nevermind, your are right it works.